src/App.tsx

```diff
@@ -1,24 +1,47 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
+import { getTodos } from './api/todos';
+import { Content } from './components/Content';
+import { Errors } from './components/Errors';
 import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+import { Error } from './types/Error';
+import { Todo } from './types/Todo';
+
+const USER_ID = 6192;
 
 export const App: React.FC = () => {
+  const [allTodos, setAllTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<Error>(Error.success);
+  const [hidden, setHidden] = useState(true);
+
+  useEffect(() => {
+    getTodos(USER_ID).then(setAllTodos);
+  }, []);
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
   return (
-    <section className="section container">
-      <p className="title is-4">
-        Copy all you need from the prev task:
-        <br />
-        <a href="https://github.com/mate-academy/react_todo-app-add-and-delete#react-todo-app-add-and-delete">React Todo App - Add and Delete</a>
-      </p>
-
-      <p className="subtitle">Styles are already copied</p>
-    </section>
+    <div className="todoapp">
+      <h1 className="todoapp__title">todos</h1>
+
+      <Content
+        todos={allTodos}
+        changeTodos={setAllTodos}
+        userId={USER_ID}
+        onError={setError}
+        onHidden={setHidden}
+      />
+
+      {!error.length || (
+        <Errors
```

Comment:
Maybe would be better to refactor it a little bit 🤔  Check if it'll work
```suggestion
      {error.length && (
        <Errors
```